README.md

```diff
@@ -1,9 +1,7 @@
 ![](images/HinodeEnhanced_v2.jpg)
-# Instrument to Instrument Translation for Solar Observations
+# Instrument to Instrument Translation (ITI) for Solar Observations
 
-# [Paper](#paper) --- [Usage](#usage) --- [Framework](#framework) 
-
-[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/github/RobertJaro/InstrumentToInstrument/blob/master/examples/ITI_translation.ipynb)
+[![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)](https://colab.research.google.com/drive/1nwXqT2SI1xqO1Vz9aYQeAadzriL7UFVM#scrollTo=k_WqPbLGYmea) [![Documentation Status](https://readthedocs.org/projects/iti-documentation/badge/?version=latest)](https://iti-documentation.readthedocs.io/en/latest/?badge=latest)
 
```

Comment:
It would be better to have the notebooks in github in order to have them archived and version tracked.

Reply:
added notebook to examples folder